bin/lint-staged.js

```diff
@@ -86,6 +88,16 @@ const options = {
 
 debug('Options parsed from command-line:', options)
 
+if (options.configPath === '-') {
+  delete options.configPath
+  try {
+    options.config = JSON.parse(fs.readFileSync(process.stdin.fd, 'utf8').toString().trim())
+  } catch {
+    console.error('Error: Could not read config from stdin.')
```

Comment:
In `lib/messages.js` you can see a collection of error and other messages. This can be moved there.

For example:
https://github.com/okonet/lint-staged/blob/78782f92b1fb0868218dc3463a3fc61dd5e70cfc/lib/messages.js#L6
https://github.com/okonet/lint-staged/blob/master/lib/messages.js#L6

Reply:
Alright, will do.